### DISCUSSION TITLE
How to solve the problem of repeating the same sentence when the audio file is temporarily silent

### DISCUSSION THREAD

**Discussion by mazecate** [openai/whisper#2052] [Q&A]
How to solve the problem of repeating the same sentence when the audio file is temporarily silent

Thanks

**Comment by Purfview** [comment]
> the problem of repeating the same sentence

Try https://github.com/openai/whisper/pull/1903 bugfix. You can install it like that:

`python -m pip install --upgrade --force-reinstall "whisper-patch-1.zip"` [ downloaded from [https://github.com/Purfview/whisper/tree/patch-1](https://github.com/Purfview/whisper/tree/patch-1) ]

If that alone doesn't help then try `hallucination_silence_threshold=2` option.
